### PR TITLE
Data import review page improvements

### PR DESCRIPTION
**Pull request by kamfosica**
Building upon PR #8 , this introduces improvement on the data import review page.

### CODE REVIEW

**Review by zikani03 — CHANGES_REQUESTED** [review]
Looks okay, just a couple of nitpicks

File: sctp-mis/src/main/resources/public/components/DataImportReview.vue [hunk 1]
```diff
@@ -0,0 +1,385 @@
...
+            <b-table-column field="districtName" label="District" v-slot="props">
+                {{ props.row.districtName }}
+            </b-table-column>
+
+            <b-table-column field="traditionalAuthorityName" label="T/A" v-slot="props">
+                {{ props.row.traditionalAuthorityName }}
+            </b-table-column>
+
+            <b-table-column field="cluserName" label="Cluster" v-slot="props">
+                {{ props.row.cluserName }}
```
[the remark this answers is not in the document]
Comment (kamfosica) [reply sctp-mis/src/main/resources/public/components/DataImportReview.vue:58]: Nice catch. Was wondering why a recent import was 100% valid but cluster came up blank

[the remark this answers is not in the document]
Comment (kamfosica) [reply sctp-mis/src/main/resources/public/components/DataImportReview.vue:58]: Nice catch. Was wondering why a recent import was 100% valid but cluster came up blank

File: sctp-mis/src/main/java/org/cga/sctp/mis/targeting/DataImportController.java [hunk 2]
```diff
@@ -69,6 +69,12 @@
 @BreadcrumbDefinition(module = ModuleNames.DATA_IMPORT, index = @BreadcrumbPath(link = "/data-import", title = "Import data from sources", navigable = true))
 public class DataImportController extends BaseController {
 
+    private static final String[] CORS_HEADERS = {"X-Is-Slice",
```
[the remark this answers is not in the document]
Comment (kamfosica) [reply sctp-mis/src/main/java/org/cga/sctp/mis/targeting/DataImportController.java:72]: Yeah. Will move into CorsConfiguration

File: sctp-mis/src/main/java/org/cga/sctp/mis/targeting/DataImportController.java [hunk 3]
```diff
@@ -148,19 +155,160 @@ ModelAndView merge(
...
+
+    @PostMapping(value = "/{import-id}/update-household-head",
+            produces = MediaType.APPLICATION_JSON_VALUE,
+            consumes = {MediaType.APPLICATION_JSON_VALUE, MediaType.APPLICATION_FORM_URLENCODED_VALUE}
+    )
+    @AdminAndStandardAccessOnly
+    public ResponseEntity<?> updateHouseholdHead(
+            @AuthenticatedUserDetails AuthenticatedUser user,
+            @PathVariable("import-id") Long importId,
+            @RequestParam("household") Long householdId,
```
[the remark this answers is not in the document]
Comment (kamfosica) [reply sctp-mis/src/main/java/org/cga/sctp/mis/targeting/DataImportController.java:276]: No problem

File: sctp-mis/src/main/java/org/cga/sctp/mis/targeting/DataImportController.java (hunk 2 above)
[the remark this answers is not in the document]
Comment (kamfosica) [reply sctp-mis/src/main/java/org/cga/sctp/mis/targeting/DataImportController.java]: Moved to config

**Review by zikani03 — APPROVED** [review]

**Pull request merged**